docs/pushmsg.md

```diff
@@ -3,7 +3,7 @@ id: pushmsg
 title: Push messages
 ---
 
-As already indicated in [Intro](index.md) you need an application to push messages to gotify/server.
+As already indicated in [Intro](index.md) you need an application to push messages to gotify/server. Only the user who created the application is able to get its notifications.
```

Comment:
In gotify/server we normally use the term message. Clients then can send notifications for received messages.
```suggestion
As already indicated in [Intro](index.md) you need an application to push messages to gotify/server. Only the user who created the application is able to see its messages.
```